src/styling-panel-property-definition.js

```diff
@@ -0,0 +1,256 @@
+const createStylingPanelDefinition = {
+  component: 'styling-panel',
+  chartTitle: 'Object.StraightTable',
+  translation: 'LayerStyleEditor.component.styling',
+  subtitle: 'LayerStyleEditor.component.styling',
+  ref: 'components',
+  useGeneral: true,
+  key: 'theme',
+  defaultValue: [],
+  items: {
+    headerSection: {
+      translation: 'properties.Header',
+      component: 'panel-section',
+      items: {
+        headerFontItem: {
+          component: 'items',
+          ref: 'components',
+          key: 'general',
+          items: {
+            headerFontWrapper: {
+              component: 'inline-wrapper',
+              items: {
+                headerFontSize: {
+                  component: 'integer',
+                  ref: 'header.fontSize',
+                  type: 'integer',
+                  width: 9,
+                  min: 5,
+                  max: 300,
+                  // placeholder: () => parseInt(styleService.getStyle('header', 'fontSize'), 10), // Not sure about placeholder, as qlik-oss do not have theme integrated in it
```

Comment:
Theme should be available through the environment. Have a look if you can access the styleService:
Theme.initializeService

Reply:
We do not get the initializeService() access from env.

Reply:
IMO this can stay commented for now, and fixed in a future PR